spec/directions/bake_chapter_glossary_spec.rb

```diff
@@ -5,8 +5,82 @@
   # TODO: test that...
   # 1) definitions with the same term sort next by the definition
 
-  xit 'works' do
+  let(:doc) do
+    Kitchen::BookDocument.new(document: Nokogiri::XML(
+      <<~HTML
+        <html>
+        <body>
+          <div data-type="chapter">
+            <div data-type='glossary'>
+              <div>
+                <dl>
+                  <dt>ZzZ</dt>
+                  <dd>Test 1</dd>
+                </dl>
+                <dl>
+                  <dt>ZzZ</dt>
+                  <dd>Achoo</dd>
+                </dl>
+                <dl>
+                  <dt>ABD</dt>
+                  <dd>Test 2</dd>
+                </dl>
+              </div>
+            </div>
+          </div>
+        </body>
+        </html>
+      HTML
+    ))
+  end
 
+  let(:metadata) do
+    new_element(
+      <<~HTML
+        <div data-type="metadata" style="display: none;">
+          <div class="authors" id="authors">Authors</div>
+          <div class="publishers" id="publishers">Publishers</div>
+          <div class="print-style" id="print-style">Print Style</div>
+          <div class="permissions" id="permissions">Permissions</div>
```

Comment:
Can you add another random `div` in here with some other class to test that it does _not_ get copied over to the glossary?

Reply:
sure